### PR TITLE
CI - Let `mvn`  execute tests in simple build GitHub workflow

### LINKED ISSUE #128: Execute unit tests in CI checks
Similarly to integration tests on OpenShift (testsuite module), tests in core (tools/) modules must be executed in the simple build CI checks workflow, so that we can monitor changes behavior in isolation at component level.

* Environment: `tools` module
* Steps to reproduce: `mvn test -pl tools/intersmash-deployments-provider,tools/intersmash-tools-provisioner -am`

### PR DESCRIPTION
**Pull request by fabiobrz**
## Description
We need to execute non integration tests as well.
Fix #128 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] Pull Request contains a description of the changes
 - [x] Pull Request does not include fixes for multiple issues/topics
 - [x] Code is self-descriptive and/or documented
 - [x] I have implemented unit tests to cover my changes
 - [x] I tested my code in OpenShift

### CODE REVIEW

File: .github/workflows/simple-build.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
       - name: Build with Maven
-        run: mvn -version ; mvn clean install -U -B -DskipTests
+        run:  mvn clean install -U -B -DskipTests ; mvn test -pl deployments/intersmash-deployments-provider,tools/intersmash-tools-core,tools/intersmash-tools-provisioners -am
```
Comment (marekkopecky) [inline .github/workflows/simple-build.yml:26]: Do we really not want to see mvn version?

Comment (fabiobrz) [reply .github/workflows/simple-build.yml:26]: Fair enough, fixing.

**Comment by marekkopecky** [comment]
Thank you, PR LGTM, I can't merge this MR now because of pending CI jobs. I will try it tomorrow.

**Pull request merged**